index.js

```diff
@@ -3,7 +3,13 @@
 
 module.exports = {
   name: 'ember-cli-airbrake',
+  isEnabled: function () {
```

Comment:
Why not just use the presence of a config value in the `config/environment.js` as the toggle? I'm not sure two flags are warranted when they will change pretty much in unison.